pkg/infrastructure/clusterapi/clusterapi.go

```diff
@@ -40,6 +41,19 @@ import (
 // interface the installer uses to call this provider.
 var _ infrastructure.Provider = (*InfraProvider)(nil)
 
+const (
+	// timeout for each provisioning step.
+	timeout = 15 * time.Minute
+
+	// infrastructure provisioning stages
```

Comment:
```suggestion
	// infrastructure provisioning stages.
```
to make the linter happy.

Reply:
doh, I forgot to delete that with my last changes. pushed.